package.json

```diff
@@ -12,7 +12,7 @@
     "run-migrations": "docker-compose up -d && serverless invoke local --function migrations",
     "revert-migration": "docker-compose up -d && ts-node ./node_modules/typeorm/cli.js migration:revert -f ./shared/config/db",
     "create-migration": "docker-compose up -d && ts-node ./node_modules/typeorm/cli.js migration:create -f ./shared/config/db -n",
-    "start-workflow": "aws stepfunctions --endpoint http://localhost:8083 start-execution --state-machine-arn arn:aws:states:us-east-1:101010101010:stateMachine:$npm_config_workflow && bash get_sf_logs.sh",
+    "start-workflow": "aws stepfunctions --endpoint http://localhost:8083 start-execution --state-machine-arn arn:aws:states:us-east-1:101010101010:stateMachine:$npm_config_workflow && docker-compose logs sfLocal",
```

Comment:
Won't it cause a problem when we run different workflows multiple times?

Reply:
We will have then a lot of logs from different workflow executions. It works in a similar way without the docker, where we are listening to all logs from the stepfunctions-local process. To solve it I think that we can provide one of two solutions:

- Add a command to package.json to check execution history with aws cli on top of executionArn. In this case we will have to copy paste the arn.
- Remove docker-compose logs from the start-workflow command, then we can add a note to README file how to listen to latest logs from the container using follow flag.

IMO, I think the best approach will be to use get-execution-history from aws-cli, but here we will lose the way how it was working before without docker - displaying all logs after running single workflow.

Reply:
What I'm afraid is we will have multiple windows for the same thing (because each execution will stream all of the logs).

I would rather go with separate logs command + readme description. It will work a little bit different than it does now (won't be a part of sls anymore) but still we will manage to have a single view on that. What do you think?

Reply:
I think that will also do the job, on the other hand when we will trigger multiple workflows at the same time it still won't be that easy to follow, but as you said we won't be streaming all of the logs with a single execution. I've introduced new changes.